src/execution_plan/ops/op_procedure_call.c

```diff
@@ -15,10 +16,8 @@ static void _yield(OpProcCall *op, SIValue *proc_output, Record r) {
 			const char *yield = op->output[i];
 			for(uint j = 0; j < array_len(proc_output); j += 2) {
 				char *key = (proc_output + j)->stringval;
-				SIValue *val = &proc_output[j + 1];
 				if(strcmp(yield, key) == 0) {
-					OpBase *base = (OpBase *)op;
-					int idx = RecordMap_LookupAlias(base->record_map, key);
+					int idx = op->op.modifies[i];
```

Comment:
Clever! Maybe some explanatory comments in this area?

Reply:
I hope this will be removed once the new hash like records will be introduced.